frontend/src/components/story/StoryDay.vue

```diff
@@ -113,6 +119,11 @@ export default {
       )
     )
   },
+  methods: {
+    updatePanelValue(day) {
+      this.$refs.panel.value = day.start.substr(0, 10)
```

Comment:
Code looks much cleaner now. Since this inconspicuous line is essential to make it work, and at the same time a little unusual (assigning a new JS property to a DOM object at runtime), would you mind adding a comment that explains this?